.travis/checklicenses_config.json

```diff
@@ -0,0 +1,14 @@
+{
+    "gopkg": "github.com/coredhcp/coredhcp",
+    "licenses": [
+        [
+            "^// Copyright 2015-2018 the CoreDHCP Authors\\. All rights reserved",
```

Comment:
nit: 2019

Reply:
bad copy-pasta, it should be 2018-present

Reply:
fixed

Reply:
fixed